test/models/patient_test.rb

```diff
@@ -126,23 +126,28 @@ class PatientTest < ActiveSupport::TestCase
       error2 = diffLineDuplicate.errors.messages[:this_phone_number_is_already_taken]
 
       assert_not_equal error1, error2
-
     end
   end
 
   describe 'pledge_summary' do
-    it "should return proper pledge summaries for various timespans" do
+    it 'should return proper pledge summaries for various timespans' do
       @patient.update appointment_date: 2.days.from_now, fund_pledge: 300
       @patient2.update appointment_date: 4.days.from_now, fund_pledge: 500,
                        pledge_sent: true, clinic: create(:clinic)
+      # this pt should be filtered out because we don't worry about resolved without fund
```

Comment:
I'm not clear in which test we're asserting this patient is filtered. Could you talk me through it?